### ISSUE TITLE
java.lang.IllegalStateException: Cannot perform this action on a sealed instance

### ISSUE THREAD

**Issue by noorumapped** [facebook/react-native#32973]
### Description

Hello I need support. I'm getting this error in pre-lauch report when submitting android mobile app  I build it on react-native and have looked everywhere on the internet but haven't found a solution. I'm not sure how to correct this error. Our application is written in react-native.

**Error in Pre-launch report:**
FATAL EXCEPTION: main
Process: com.nuba.mobile.android, PID: 18007
java.lang.IllegalStateException: Cannot perform this action on a sealed instance.
	at android.view.accessibility.AccessibilityNodeInfo.enforceNotSealed(AccessibilityNodeInfo.java:2992)
	at android.view.accessibility.AccessibilityNodeInfo.setSource(AccessibilityNodeInfo.java:818)
	at android.view.accessibility.AccessibilityNodeInfo.setSource(AccessibilityNodeInfo.java:795)
	at android.view.accessibility.AccessibilityNodeInfo.obtain(AccessibilityNodeInfo.java:3008)
	at android.view.View.createAccessibilityNodeInfoInternal(View.java:7897)
	at android.view.View.createAccessibilityNodeInfo(View.java:7883)
	at android.view.AccessibilityInteractionController$AccessibilityNodePrefetcher.prefetchDescendantsOfRealNode(AccessibilityInteractionController.java:1051)
	at android.view.AccessibilityInteractionController$AccessibilityNodePrefetcher.prefetchAccessibilityNodeInfos(AccessibilityInteractionController.java:876)
	at android.view.AccessibilityInteractionController.findAccessibilityNodeInfoByAccessibilityIdUiThread(AccessibilityInteractionController.java:199)
	at android.view.AccessibilityInteractionController.-wrap1(Unknown Source:0)
	at android.view.AccessibilityInteractionController$PrivateHandler.handleMessage(AccessibilityInteractionController.java:1225)
	at android.os.Handler.dispatchMessage(Handler.java:105)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6938)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)

**They tested it on this device for example.**
<img width="980" alt="Screen Shot 2022-01-26 at 4 15 52 PM" src="https://user-images.githubusercontent.com/86007357/151248205-11e8e887-a644-462d-870f-3c56dcf5e31a.png">

**I have attached the logcat**
[logcat-2.txt](https://github.com/AndBible/and-bible/files/7945662/logcat-2.txt)

### Version

0.59.10

### Output of `npx react-native info`

this is a pre-launch issue

### Steps to reproduce

in the description

**Comment by diegorochawebdg** [comment]
I'm facing the same issue with Samsung Galaxy S9 and Xiaomi Redmi 6A in the pre-lauch report
![image](https://user-images.githubusercontent.com/6142884/153591229-c86213b4-d531-4319-8ed6-4684d45e63dd.png)

**Comment by palanpuricoder** [comment]
getting same error for same devices ,

**Comment by blavalla** [comment]
Do you have any reproduction steps to trigger this error? This could be caused by a lot of different things unfortunately.
